static/js/comfortmodels.js

```diff
@@ -97,7 +97,7 @@ comf.go = function(){
     return r;
 }
 
-comf.still_air_threshold = 0.1; // m/s
+comf.still_air_threshold = 0.2; // m/s
```

Comment:
This is a major change... I don't see anything about in the description, so I need to make sure this isn't a mistake. We're changing the still air threshold? Is there anything in the standard about this value these days?

I wouldn't want to deploy this unless we confirm from Ed and Stefano. We'll probably get lots of emails about it otherwise :)